components/prism-java.js

```diff
@@ -55,7 +55,9 @@
 		'operator': {
 			pattern: /(^|[^.])(?:<<=?|>>>?=?|->|--|\+\+|&&|\|\||::|[?:~]|[-+*/%&|^!=<>]=?)/m,
 			lookbehind: true
-		}
+		},
+		'constant': /\b[A-Z][A-Z_\d]{2,}\b/,
```

Comment:
Why did you require constants to have at least 3 characters? There are a few English words with 2 characters (e.g. "no") that might be used as static final fields or enum variants.

Reply:
I changed it to 2 characters